tests/testthat/helper-testing.R

```diff
@@ -49,19 +66,36 @@ rmot_test_single_individual <- function(model_name,
 }
 
 rmot_test_multi_individual <- function(model_name, data, est_dim){
-  # Test constant multi-individual
-  suppressWarnings( #Suppresses stan warnings
-    multi_ind_test <- rmot_model(paste0(model_name, "_multi_ind")) |>
-      rmot_assign_data(n_obs = data$n_obs, #integer
-                       n_ind = data$n_ind, #integer
-                       y_obs = data$y_obs, #vector length N_obs
-                       obs_index = data$obs_index, #vector length N_obs
-                       time = data$time, #Vector length N_obs
-                       ind_id = data$ind_id, #Vector length N_obs
-                       y_0_obs = data$y_0_obs #vector length N_ind
-      ) |>
-      rmot_run(chains = 2, iter = 100, verbose = FALSE, show_messages = FALSE)
-  )
+  if(! is.null(data$step_size)){
+    # Test multi-individual
+    suppressWarnings( #Suppresses stan warnings
+      multi_ind_test <- rmot_model(paste0(model_name, "_multi_ind")) |>
+        rmot_assign_data(step_size = data$step_size, #real
+                         n_obs = data$n_obs, #integer
+                         n_ind = data$n_ind, #integer
+                         y_obs = data$y_obs, #vector length N_obs
+                         obs_index = data$obs_index, #vector length N_obs
+                         time = data$time, #Vector length N_obs
+                         ind_id = data$ind_id, #Vector length N_obs
+                         y_0_obs = data$y_0_obs #vector length N_ind
+        ) |>
+        rmot_run(chains = 2, iter = 100, verbose = FALSE, show_messages = FALSE)
```

Comment:
2 chains?

Reply:
Yeah this is just testing that multiple chains produce the right size of samples.